Sources/Orbit/Support/TextConcatenation/DynamicTextWrapper.swift

```diff
@@ -0,0 +1,52 @@
+import SwiftUI
+
+struct DynamicTextWrapper: View {
```

Comment:
Suggestion: `ConcatenatedTextWrapper` ? `Dynamic` is a vague term.

Reply:
Renamed to `ConcatenatedText`